docs/customization/model.rst

```diff
@@ -99,7 +108,9 @@ Under the ``sylius_*`` where ``*`` is the name of the bundle of the model you ar
         resources:
             country:
                 classes:
-                    model: App\Entity\Country
+                    model: App\Entity\Addressing\Country
```

Comment:
We are missing the interface config I suppose?

Reply:
This is a problem because if we add an interface now we'll get errors...

Reply:
What do you mean by errors?

Reply:
This working only if I add 
resolve_target_entities:
            Sylius\Component\Addressing\Model\CountryInterface: App\Entity\Addressing\Country

in config/packages/doctrine.yml

Its ok?

Reply:
cc @pamil could you take a look? :D At first I proposed the `resolve_target_entities` solution but I believe it's not the most elegant one.